feat: serverless ip firewall for public endpoint (#20494)
pingcap/docs: tidb-cloud/connect-via-standard-connection-serverless.md

    > **Note:**
    >
    > When you connect to a TiDB Cloud Serverless cluster, you must include the prefix for your cluster in the user name and wrap the name with quotation marks. For more information, see [User name prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix).

>     > Your client IP must be in the allowed IP rules of the public endpoint of your cluster. For more information, see [Connect to TiDB Cloud Serverless via Public Endpoint](/tidb-cloud/networking-public.md)

Reviewer: ```suggestion
    > Your client IP must be in the allowed IP rules of the public endpoint of your cluster. For more information, see [Connect to TiDB Cloud Serverless via Public Endpoint](/tidb-cloud/set-up-public-endpoint-connections-serverless.md)
```

>     > Your client IP must be in the allowed IP rules of the public endpoint of your cluster. For more information, see [Connect to TiDB Cloud Serverless via Public Endpoint](/tidb-cloud/set-up-public-endpoint-connections-serverless.md)

Reviewer 2: ```suggestion
    > Your client IP must be in the allowed IP rules of the public endpoint of your cluster. For more information, see [Configure TiDB Cloud Serverless Firewall Rules for Public Endpoints](/tidb-cloud/configure-serverless-firewall-rules-for-public-endpoints.md).
```
